src/sql/workbench/common/styler.ts

```diff
@@ -49,3 +55,34 @@ export function attachTabbedPanelStyler(widget: IThemable, themeService: IThemeS
 		activeTabContrastBorder: cr.activeContrastBorder
 	}, widget);
 }
+
+export function attachCalloutDialogStyler(widget: IThemable, themeService: IThemeService, style?: IModalDialogStyleOverrides): IDisposable {
+	return attachStyler(themeService, {
+		dialogForeground: (style && style.dialogForeground) || sqlcr.calloutDialogForeground,
+		dialogHeaderAndFooterBackground: (style && style.dialogHeaderAndFooterBackground) || sqlcr.calloutDialogHeaderFooterBackground,
+		dialogBodyBackground: (style && style.dialogBodyBackground) || sqlcr.calloutDialogBodyBackground,
+		dialogInteriorBorder: (style && style.dialogInteriorBorder) || sqlcr.calloutDialogInteriorBorder,
+		dialogExteriorBorder: (style && style.dialogExteriorBorder) || sqlcr.calloutDialogExteriorBorder,
+		dialogShadowColor: (style && style.dialogShadowColor) || sqlcr.calloutDialogShadowColor
+	} as IModalDialogStyleOverrides, widget);
+}
+
+export function attachCustomDialogStyler(widget: IThemable, themeService: IThemeService, dialogStyle?: string, style?: IModalDialogStyleOverrides): IDisposable {
+	if (dialogStyle === 'callout') {
+		return attachStyler(themeService, {
```

Comment:
Just call `attachCalloutDialogStyler` and `attachModalDialogStyler` here instead